SuperGoodSoft/solidus_taxjar
# Use nexus regions for taxable address checks

Closes #57.

What is the goal of this PR?
---

To meet a TaxJar certification requirement:

> Only send a request to v2/taxes when a merchant has nexus.

How do you manually test these changes? (if applicable)
---

Not applicable. I've recorded TaxJar API requests that confirm that we're doing the right thing. Unless some of the United States change their state abbreviations, or a new U.S. state  appears and we need to upgrade the `carmen` gem, this should just work.

Merge Checklist
---
